### PR TITLE
Adding layout/plan.h to reading-list.md [WIP]

### PR DESCRIPTION
**Pull request by TechWriter-Mayur**
Added layout plan information in the function shipping as requested in 

https://github.com/Seagate/m0client-sample-apps/issues/12#issuecomment-771828619
-----
[View rendered doc/reading-list.md](https://github.com/TechWriter-Mayur/cortx-motr/blob/main/doc/reading-list.md)

### CODE REVIEW

**Review by johnbent — APPROVED** [review]
FYI @nikitadanilov and @andriytk

File: doc/reading-list.md [hunk 1]
```diff
@@ -86,6 +86,8 @@ Function Shipping
 
 * Function shipping (In-storage-compute) usage in motr: [PDF](PDF/motr_function_shipping.pdf).
 
+  * The layout plan interface allow a generic and fault-tolerant implementation of function shipping: [layout plan](https://github.com/Seagate/cortx-motr/blob/main/layout/layout.h)
+
```
Comment (huanghua78) [inline doc/reading-list.md:90]: The doc is at: 
http://gerrit.mero.colo.seagate.com/plugins/gitiles/mero/+/refs/heads/dev/layout-plan/layout/plan.h

It is seagate internal site, isn't it?
And this is on a development branch.

I don't think it is accessible from outside.

File: doc/reading-list.md [hunk 2]
```diff
@@ -86,6 +86,8 @@ Function Shipping
 
 * Function shipping (In-storage-compute) usage in motr: [PDF](PDF/motr_function_shipping.pdf).
 
+  * The layout plan interface allow a generic and fault-tolerant implementation of function shipping: [layout plan](https://github.com/Seagate/cortx-motr/blob/main/layout/layout.h)
```
Comment (huanghua78) [inline doc/reading-list.md:89]: s
/The layout plan interface allow 
/The layout plan interface allows

File: doc/reading-list.md (hunk 1 above)
Comment (TechWriter-Mayur) [reply doc/reading-list.md:90]: The same document is available at GitHub on the following link: 
https://github.com/Seagate/cortx-motr/blob/main/layout/layout.h

I have created the GitHub link so everyone can access the link.

Comment (huanghua78) [reply doc/reading-list.md:90]: Are you sure these two files are the same? Or the contents are similar?
http://gerrit.mero.colo.seagate.com/plugins/gitiles/mero/+/refs/heads/dev/layout-plan/layout/plan.h
https://github.com/Seagate/cortx-motr/blob/main/layout/layout.h

I don't think so.

File: doc/reading-list.md [hunk 3]
```diff
@@ -86,6 +86,8 @@ Function Shipping
 
 * Function shipping (In-storage-compute) usage in motr: [PDF](PDF/motr_function_shipping.pdf).
 
+  * The layout plan interface allows a generic and fault-tolerant implementation of function shipping: [layout plan](https://github.com/Seagate/cortx-motr/blob/main/layout/layout.h)
```
Comment (johnbent) [inline doc/reading-list.md:89]: @TechWriter-Mayur , there should be a way to make this a relative (i.e. not absolute) URL.  Can you please fix it to be relative?

**Review by johnbent — CHANGES_REQUESTED** [review]

File: doc/reading-list.md (hunk 1 above)
Comment (johnbent) [reply doc/reading-list.md:90]: @huanghua78 , I think the only difference is that Mayur changed the license text to Apache.  @TechWriter-Mayur , is that correct?

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

**Comment by huanghua78** [comment]
This is closed. When #411 is done, we may re-open it.

**Pull request closed without merging**